diff -ru --no-dereference ../glib-2.66.8/meson.build ./meson.build
--- glib-orig/meson.build	2021-03-18 21:47:48.374693600 +0800
+++ glib/meson.build	2026-07-01 19:52:41.396558700 +0800
@@ -1828,9 +1828,9 @@
   if cc.has_header_symbol('pthread.h', 'pthread_condattr_setclock')
     glib_conf.set('HAVE_PTHREAD_CONDATTR_SETCLOCK', 1)
   endif
-  if cc.has_header_symbol('pthread.h', 'pthread_cond_timedwait_relative_np')
-    glib_conf.set('HAVE_PTHREAD_COND_TIMEDWAIT_RELATIVE_NP', 1)
-  endif
+  #if cc.has_header_symbol('pthread.h', 'pthread_cond_timedwait_relative_np')
+  #  glib_conf.set('HAVE_PTHREAD_COND_TIMEDWAIT_RELATIVE_NP', 1)
+  #endif
   if cc.has_header_symbol('pthread.h', 'pthread_getname_np', prefix : pthread_prefix)
     glib_conf.set('HAVE_PTHREAD_GETNAME_NP', 1)
   endif
